docs/csharp/event-pattern.md

```diff
@@ -173,18 +142,7 @@ indicate a cancel incorrectly.
 Let's implement the first version for this sample. You need to add a
 boolean field to the FileFoundEventArgs type:
```

Comment:
the code doesn't show the FileFoundEventArgs type, it shouws FileFoundArgs. There are multiple references in throughout to FileFoundArgs.
Also, it would be preferable to name the field: you need to add a Boolean field named \`CancelRequested\`...